src/cppcache/src/TcrMessage.hpp

```diff
@@ -171,7 +173,8 @@ class CPPCACHE_EXPORT TcrMessage {
     GET_DURABLE_CQS_DATA_ERROR = 106,
     GET_ALL_WITH_CALLBACK = 107,
     PUT_ALL_WITH_CALLBACK = 108,
-    REMOVE_ALL = 109
+	REMOVE_ALL = 109,
+	HANDSHAKE = 110
```

Comment:
These numbers correspond to protocol message numbers on the server. We can just add one here and expect it not cause issues later.